lib/cmxl/fields/floor_limit_indicator.rb

```diff
@@ -0,0 +1,20 @@
+module Cmxl
+  module Fields
+    class FloorLimitIndicator < Field
+      self.tag = 34
+      self.parser = /(?<currency>[a-zA-Z]{3})(?<type_indicator>[DC]?)(?<amount>[\d|,|\.]{4,15})/i
+
+      def credit?
+        data['type_indicator'].empty? || data['type_indicator'] == 'C'
```

Comment:
what is the case if `type_indigator` is empty? when does this happen? - right now both `credit?` and `debit?` would return true, no?

Reply:
that's indeed the idea behind it. if indicator is not provided that means the provided limit is for both types

Reply:
the floor limit indicator is not used by the statements in any way as every single example of an mt942 implementation by a bank statet, that the floor limit indicator is not used and is always returned as 0.00, thus rendering this field useless

Reply:
interesting. thx.